### PR TITLE
Fixed some typos

### PR DESCRIPTION
**Pull request by JacobCZ**

### CODE REVIEW

File: source/plugin/project/version-numbers.rst [hunk 1]
```diff
@@ -3,16 +3,15 @@ Version Numbers
...
-a ``.`` and additional label appended with a hyphens ``-`` such as ``3.1`` or ``1.0.3-API7`` similar to SemVer.
+Sponge recommends using a version identifier that consists of either two or three numeric version parts separated with
+a ``.`` and additional label appended with a hyphen ``-`` such as ``3.1`` or ``1.0.3-API7`` similar to SemVer.
 Start with ``1.x`` if your plugin is ready. Start with ``0.x`` if your plugin is not yet ready or unstable.
 
 .. note::
 
-    While Sponge recommends using the above versioning scheme, plugin authors are free to choose their own. Please keep
-    in mind that your users have to deal with a multitude of versioning schemes, so keep it simple.
+    While Sponge recommends using the above versioning scheme, plugin authors are free to choose their own. Please keep in mind that your users have to deal with a multitude of versioning schemes, so keep it simple.
```
Comment (ST-DDT) [inline source/plugin/project/version-numbers.rst]: Please keep the line length at 120 characters:

https://docs.spongepowered.org/stable/en/contributing/spongedocs.html?highlight=maximum%20length#style-guide

**Review by ST-DDT — CHANGES_REQUESTED** [review]
Thanks for your contribution.

File: source/plugin/project/version-numbers.rst [hunk 2]
```diff
@@ -23,13 +22,12 @@ the ``MAJOR``, ``MINOR`` and ``PATCH`` version parts.
...
-* A change in **MINOR** version indicates that a feature has been added without breaking backwards
-  compatibility.
+* A change in **MINOR** version indicates that a feature has been added without breaking backward compatibility.
 * A change in the **PATCH** version indicates that some bugs have been fixed.
 
-The labels can be used to mark builds as pre-releases or contain build metadata such as the supported minecraft version.
+The labels can be used to mark builds as pre-releases or contain build metadata such as the supported Minecraft version.
 If you increase one part of the version, then you will set the following parts back to zero. Using this strategy you
-can find the latest version by comparing the version blocks with each other, however this scheme does not contain any
+can find the latest version by comparing the version blocks with each other, however, this scheme does not contain any
```
Comment (Inscrutable) [inline source/plugin/project/version-numbers.rst]: The sentence works better if the comma after "other" becomes a semicolon.

File: source/plugin/project/version-numbers.rst [hunk 3]
```diff
@@ -58,7 +56,7 @@ Timestamp
 
 This is a less commonly used versioning scheme, but it is also a very simple one, as it does not contain any additional
 information in the version numbers. It does not matter which separators you use (if any) or which date and time parts
-you include. However we recommend you to order date and time parts in the descending order of duration like
+you include. However, we recommend you to order date and time parts in the descending order of duration like
```
Comment (Inscrutable) [inline source/plugin/project/version-numbers.rst]: Whilst we're quibbling this sentence, I'd replace "you to order" with "ordering" (or "that you").
The "the" in front of "descending" isn't needed, too.

**Review by ST-DDT — APPROVED** [review]

File: source/plugin/project/version-numbers.rst [hunk 4]
```diff
@@ -58,9 +57,9 @@ Timestamp
 
 This is a less commonly used versioning scheme, but it is also a very simple one, as it does not contain any additional
 information in the version numbers. It does not matter which separators you use (if any) or which date and time parts
-you include. However we recommend you to order date and time parts in the descending order of duration like
-``YYYY-MM-DD``. If you sort your files alphabetically, please make sure that the order would be the same as the version
-order. It is also possible to add a build qualifier or a build number.
+you include. However, we recommend ordering date and time parts in descending order of duration like ``YYYY-MM-DD``. 
+If you sort your files alphabetically, please make sure that the order would be the same as the versionorder. It is
```
Comment (Inscrutable) [inline source/plugin/project/version-numbers.rst]: Why is "versionorder" one word now?

Comment (JacobCZ) [reply source/plugin/project/version-numbers.rst]: Because fuck VIM :stuck_out_tongue: Lemme fix that real quick...

**Pull request merged**